cfgmgr/intfmgr.cpp

```diff
@@ -74,10 +101,12 @@ IntfMgr::IntfMgr(DBConnector *cfgDb, DBConnector *appDb, DBConnector *stateDb, c
     {
        mySwitchType = swtype;
     }
+
+    g_ipv6Flag = inferIPV6Capability();
 }
 
 void IntfMgr::setIntfIp(const string &alias, const string &opCmd,
-                        const IpPrefix &ipPrefix)
+                        const IpPrefix &ipPrefix, bool retryV6)
```

Comment:
I don't  see any retry logic, why this flag is called retryV6?

Reply:
I'm recursively calling this function again  (and also enabling ipv6 on the iface) if this boolean is set. Will rename the argument.